### DISCUSSION TITLE
How can I upload images to s3 using HTTP2 protocol?

### DISCUSSION THREAD

**Discussion by hakan-bilgin** [aws/aws-sdk-js-v3#2739] [General]
I want to try HTTP2 for big amount of concurrent image uploads and see performance difference? How can I force h2 protocol as a Js client in browser environment? I couldn't succeed by using FetchHttpHandler.

**Comment by nicolae-olariu** [comment]
@hakan-bilgin Did you find any solution for this?

**Reply by hakan-bilgin** [reply]
Unfortunately, I couldn't find at that time. I was working for another company at that time and I didn't need this again. I remember that it was the AWS servers which didn't accept HTTP2 protocol. I also remember comments telling that HTTP2 can be even less performant in some cases. I don't know if AWS started to support this during the last two years.
